repose-aggregator/core/repose-core-api/src/test/scala/org/openrepose/core/systemmodel/config/SystemModelSchemaTest.scala

```diff
@@ -180,19 +180,6 @@ class SystemModelSchemaTest extends ConfigurationTest {
       validator.validateConfigString(config)
     }
 
-    it("should reject the config if there are no defined destination endpoints nor targets") {
```

Comment:
Why was this test removed?

Reply:
Since getting rid of clusters means that we can also get rid of `target` destinations we no longer need an `assert` to verify that at least one `endpoint` or `target`. Instead, that requirement is part of the schema itself (encoded as the `minOccurs="1"` attribute and value).

Reply:
We normally test schema assertions/expectations just to ensure they aren't accidentally changed in the future.